### ISSUE TITLE
Date value in the Date Index is  stored as a string.

### ISSUE THREAD

**Issue by vlmercado** [ranaroussi/fix-yahoo-finance#1]
The Date column/index of the dataframe is a dtype: object. The dataframe created in the previously functioning pandas_datareader was a dtype: datetime64[ns]. To maintain backward compatilitbity, I added:

```
df.reset_index(inplace=True)
df['Date'] = pd.to_datetime(df['Date'])
```

**Comment by ranaroussi** [comment]
I've fixed that in version 0.0.6 a few hours before you posted this issue :)

Please upgrade and let me know if this is still an issue.

**Comment by ranaroussi** [comment]
closing the issue for now. please reopen if the problem shows up again
